source/atlas-data-lake-testing/tests/unified/getMore.yml

```diff
@@ -0,0 +1,48 @@
+description: "getMore"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name test
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name driverdata
+
+tests:
+  - description: "A successful find event with getMore"
+    operations:
+      - object: *collection0
+        name: find
+        arguments:
+          filter: { a: { $gte: 2 } }
+          sort: { a: 1 }
+          batchSize: 3
+          limit: 4
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collection0Name
+                filter: { a: { $gte : 2 } }
+                sort: { a: 1 }
+                batchSize: 3
+                limit: 4
+              commandName: find
+              databaseName: *database0Name
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: [ int, long ] }
+                # collection name will be an internal identifier
+                collection: { $$type: string }
```

Comment:
The original tests didn't assert `getMore` or `collection` at all, but we can now use `$$type`.